src/2d/shallow/surge/model_storm_module.f90

```diff
@@ -407,7 +407,7 @@ subroutine set_holland_1980_fields(maux, mbc, mx, my, xlower, ylower,    &
         ! Local storage
         real(kind=8) :: x, y, r, theta, sloc(2), B
         real(kind=8) :: f, mwr, mws, Pc, Pa, dp, wind, tv(2), radius
-        real(kind=8) :: mod_mws, trans_speed, ramp
+        real(kind=8) :: mod_mws, trans_speed, ramp, TransSpdX, TransSpdY
```

Comment:
Do we need `trans_speed` anymore?  Also can you switch to the python style for variables rather than camelcase?

Reply:
fixed!